### ISSUE TITLE
Value of header "To: Undisclosed recipients:;" is missing space ("Undisclosedrecipients")

### ISSUE THREAD

**Issue by fundo-moretti** [zbateson/mail-mime-parser#241]
It's probably not very important, but there's an inconsistency between version 2.x and 3.x concerning the `To` header when it has the value `"Undisclosed recipients:;"`.

Source:
```
From: Test <test@example.com>
Subject: Subject
To: Undisclosed recipients:;
```

Code:
```
$mailParser = new MailMimeParser();
$message = $mailParser->parse($source, false);
$headerTo = $message->getHeader('To');
echo $headerTo->getParts()[0]->getValue()
```
Output in release 2.4.1: `"Undisclosed recipients"`
Output in release 3.0.2: `"Undisclosedrecipients"`

Attached, dump of `$headerTo->getParts()` for both releases:
[dump.txt](https://github.com/user-attachments/files/16502993/dump.txt)

**Comment by zbateson** [comment]
Good find, that one could've lingered, thank you!

**Comment by zbateson** [comment]
Released in 3.0.3, let me know if you have any issues.
